scripts/rename_project.sh

```diff
@@ -3,4 +3,14 @@
 mv python_template "$1"
 mv python_template.deps.yaml "$1".deps.yaml
 
+# change project name in all files
 find . \( -type d -name .git -prune \) -o \( -type f -not -name 'tasks.json' -not -name 'update_from_template.sh' -not -name 'update_from_template_ours.sh' \) -print0 | xargs -0 sed -i "s/python_template/$1/g"
+
+# author name
```

Comment:
What would happen if you ran this script again, i.e. after it had already been renamed once?
What happens if you run the script from the cmd line and don't pass in all the args?

lastly, I noticed there is a bug in the original rename code (which you've now copied).  "update_from_template_ours.sh" has been removed from the project so should be removed from all

Reply:
This script will only work once, and I assume it is meant to be only used once when creating a project off of this template.
I did not consider the situation where the script will be run from the commandline because the original code did not seem to consider it either? I assumed the use case is only via vscode tasks.

Reply:
Ok cool.  I generally have been using it from the task, but I have also been trying to reduce the number/complexity of the files in the template so that I can copy just 1 or 2 files into another repo and use them there.   For example, I have been copying launch_vscode.sh, project.deps.yaml and install_pixi.sh into other projects and I get most of the template features without changing the project structure too much (but no tasks). 

I was not copying the rename script as its a bit more specific so maybe its fine in this case.